src/table/utils/handle-click.ts

```diff
@@ -37,8 +37,11 @@ export const handleClickToSort = (
   changeSortOrder: ChangeSortOrder,
   isInteractionEnabled: boolean
 ) => {
-  !(evt.target as HTMLElement).getElementsByClassName('sn-table-head-menu-item-button')[0]?.ariaDisabled &&
-    !(evt.target as HTMLElement).closest('#sn-table-head-menu-button') &&
+  const target = evt.target as HTMLElement;
+  !target.closest('#sn-table-head-menu-button') &&
```

Comment:
this works but not sure we need all of them. But this function  wont be used when we move the click listener from the cell to the button inside the cell